webapp/src/app.tsx

```diff
@@ -5613,6 +5613,11 @@ document.addEventListener("DOMContentLoaded", async () => {
         await auth.loginCallbackAsync(query);
     }
 
+    // Handle logins that start via query parameter (for now, this is only Clever)
+    if (query["authlogin"]) {
```

Comment:
That was the goal. I ended up reverting that portion because of a potential runtime issue described here: https://github.com/microsoft/pxt/pull/9458#discussion_r1163333923

I'd like to do that check in a more elegant way than "if (query["authlogin"] && query["authlogin"] === "clever" || query["authlogin"] === "microsoft" ...). I could also just create an array in this file.